test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -15,6 +15,7 @@ public void isAnyNull() {
         assertFalse(Utils.isAnyNull());
 
         // Any non-empty list
+        assertFalse(Utils.isAnyNull(new Object()));
```

Comment:
We expected you to write test case methods separately for this exercise. 
Note the naming convention for test methods (the first part is feature/method under test), second is the testing scenario and the third is the expected behavior. 
E.g. isAnyNull_emptyList_falseReturned
Part 2 and 3 of the test name can be removed to indicate an increased test scope